doc/guides/maintaining-npm.md

```diff
@@ -4,10 +4,6 @@ New pull requests should be opened when a "next" version of npm has
 been released. Once the "next" version has been promoted to "latest"
 the PR should be updated as necessary.
 
-One week after the "latest" release has been promoted, it can land on master
-assuming no major regressions are found. There are no additional constraints
-for Semver-Major releases.
-
```

Comment:
@Trott - what would be an advantage of doing this? IMO a one week grace period is justified as a resiliency and quality measure.

Reply:
@gireeshpunathil I think it makes sense to withhold it from releases, but I'm less certain that there's much advantage to delaying it landing on master.  There was considerable discussion about this in https://github.com/nodejs/node/pull/29922. In that discussion, there seem to be a lot of people in favor of getting rid of the delay and treating npm PRs like other PRs (especially if you include emoji reactions).  But I'm totally OK if someone thinks this goes too far and blocks this change. Ultimately, I'm comfortable with changing it because we can always change the rule back.